app/controllers/v0/veteran_readiness_employment_claims_controller.rb

```diff
@@ -33,5 +33,20 @@ def filtered_params
     def short_name
       'veteran_readiness_employment_claim'
     end
+
+    def encrypted_user(user)
+      user_struct = OpenStruct.new(
+        participant_id: user.participant_id,
+        pid: user.participant_id,
+        edipi: user.edipi,
+        vet360ID: user.vet360_id,
+        dob: user.birth_date,
+        ssn: user.ssn,
+        loa3: user.loa3?,
+        uuid: user.uuid,
+        va_profile_email: user&.va_profile_email
```

Comment:
I think you can safely (pun intended) you remove the safe navigation operator.

Reply:
done!